src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOverridingAndHiding.cs

```diff
@@ -1155,11 +1160,17 @@ public static void Main()
             // although it may change from one version of the CLR to another (not sure). If it turns out that this makes
             // the test flaky, we can delete this test.
 
+            // Note: When implementing covariant return types, we caused the compiler to insert a methodimpl for the
+            // override that forces the runtime to agree with the compile-time override behavior.  It is therefore no
+            // longer ambiguous from the runtime's perspective.  The output of this test is therefore correct from a
+            // language perspective and there is no ambiguity.
             var comp = CompileAndVerify(source, expectedOutput: @"
+Base.Method(out, ref)
 Derived.Method(ref, out)
-Base.Method(ref, out)
-Base.Method(ref)");
+Derived.Method(ref)
+Base2.Method(out)");
```

Comment:
>"); [](start = 17, length = 3)

Minor suggestion: consider putting ```");``` on the next line. This way changes in the base-line will be easier to review in the future.